packages/cli/src/utils.ts

```diff
@@ -197,6 +203,41 @@ export function tsTemplate(
   }\n//# sourceMappingURL=${filename}.map\n`;
 }
 
+function isValidJsIdentifier(name: string) {
+  return /^(?!\d)[\w$]+$/.test(name);
+}
+
+export function mjsTemplate(exports: string[], relativePath: string) {
+  const escapedPath = JSON.stringify(relativePath);
+  return `import * as _ns from ${escapedPath};\n${exports
+    .map((name, i) => {
+      if (name === "default") {
+        return `export default _ns.default.default;`;
+      }
+      if (!isValidJsIdentifier(name)) {
+        const escapedName = JSON.stringify(name);
+        return `var _export${i} = _ns[${escapedName}];\nexport { _export${i} as ${escapedName} };`;
+      }
```

Comment:
Rollup doesn't actually handle this today too well either, I opened a fix for this [here](https://github.com/rollup/rollup/pull/4939)

Reply:
Again, similar to the types, this wouldn't work as written since the `default` isn't in the `import * as`. This should just be `export *` + stuff for the default.

Reply:
hm, we don't 🤔 this is in the `!isValidJsIdentifier` block. However, like you suggested - most of it can probably be simplified with `export *`. Unless you think now there is some reason to keep the current "explicit" strategy?

Reply:
Oh sorry, I missed the `if`